.evergreen/config.yml

```diff
@@ -412,6 +412,12 @@ functions:
           # Don't use ${file} syntax here because evergreen treats it as an empty expansion.
           [ -f "$file" ] && sh $file || echo "$file not available, skipping"
 
+  "decrypt atlas URI list":
+    - command: shell.exec
+      params:
+        script: |
+          openssl aes-256-cbc -S "${encrypted_uris_salt}" -K "${encrypted_uris_key}" -iv "${encrypted_uris_iv}" -in ${PROJECT_DIRECTORY}/.evergreen/atlas-uris.txt.enc -out ${PROJECT_DIRECTORY}/.evergreen/atlas-uris.txt -d
```

Comment:
Are the `encrypted_uris_*` set as environment variables through Evergreen?

Reply:
That is correct, they are set in the project settings.